apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```diff
@@ -48,6 +49,16 @@ const CurriculumCatalog = ({
     setAssignSuccessMessage('');
   };
 
+  const getSelfPacedPlPath = selfPacedPlId => {
+    if (selfPacedPlId !== null) {
+      const course_offering = selfPacedPlCourseOfferings.find(
+        curriculum => curriculum.id === selfPacedPlId
+      );
+      return course_offering ? course_offering.course_version_path : null;
+    }
+    return null;
+  };
+
```

Comment:
If `selfPacedPlCourseOfferings` were already part of their respective course offerings within `curriculaData`, I don't think we would need this function?

Reply:
Sorry I feel like we discussed this before so forgive me if you've already answered this. Is there a reason why we can't obtain this information within the `course_offering.rb` model so that a given course_offering has its related self-paced PL ids handy?

Reply:
Sorry Turner this is my fault, so the problem was the self paced professional learning course offerings are not included in the curriculaData so i thought i had to include these courses in the script. You're right i could instead use the association we created and include that in the summary method. Ill make this change. Thank you!

Reply:
No worries! Sounds good :)